### PR TITLE
added-swagger-to-project

### LINKED ISSUE #20: [Swagger] Add Swagger to the project
## Description
- install required npm packages
- in initialization.js file indicate swagger options and set them as settings
- create Swagger UI docs page
- add docs folder for API documentation
## Acceptance Criteria
- [ ] Swagger should be connected to the project and the documentation page should exist

### PR DESCRIPTION
**Pull request by grbvtsk**
src/docs/paths/*yaml files-  **contain HTTP's methods documentation.** 
src/docs/swagger.js - **the main file of API documentation.**
**in initialization.js file indicated swagger**

Well, for creating ALL HTTP's methods documentation - I create tasc [Swagger] Get over API documentation #38

_To start swagger:_
**1) npm start**
**2 ) use this url :http://localhost:8080/api-docs/**
image

### CODE REVIEW

File: src/initialization/initialization.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const express = require('express')
 const cors = require('cors')
 const cookieParser = require('cookie-parser')
+const swaggerui = require('swagger-ui-express')
+const swaggerDocs = require('../docs/swagger')
```
Comment (vlad-khrychov) [inline src/initialization/initialization.js]: use alias for path, 
```suggestion
const swaggerDocs = require('~/docs/swagger')
```

**Review by vlad-khrychov — COMMENTED** [review]
Did you added the new task about filling the documentation on our board? If yes you should write about it in description.

**Review by olehpona — APPROVED** [review]
Works well

**Comment by grbvtsk** [comment]
> Did you added the new task about filling the documentation on our board? If yes you should write about it in description.

Ok , I just did it. Thanks

**Review by SachkovaYuliia — APPROVED** [review]

**Review by tufned — APPROVED** [review]
works on my machine)

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**